### ISSUE TITLE
Nominatim Ansible role

### ISSUE THREAD

**Issue by fdv** [openstreetmap/Nominatim#686]
Hello,

Not sure how to put this, so I'm opening an issue.

My company has released an Ansible role to deploy Nominatim on a Debian server. Everything is there. https://github.com/synthesio/infra-ansible-nominatim

Hope it might help some people here.

**Comment by mtmail** [comment]
Awesome! And great it installs a stable (from http://www.nominatim.org/) release and not direct from git master branch.

We need to create a section in our documentation for these. There's at least three docker images now, also two chef recipes and vagrant instructions.

**Comment by lonvia** [comment]
Yes, we should add a section like that. And I agree that one based on a release. However, we are in a bit of a difficult situation right now. There are significant changes in the installation procedure between master and the release and I hope to get a new release out soon. So it would be good to have something that supports the master setup as well.

**Comment by Firefishy** [comment]
@fdv @mtmail How about a PR to install notes or similar to include a link to the ansible role?

**Comment by mtmail** [comment]
@Firefishy Thanks for pushing. I want to move the (supported, but legacy) 2.5 install instructions from wiki.openstreetmap.org to http://nominatim.org/release-docs/latest/ this week and will add links to ansible and docker options as well.
